README.md

```diff
@@ -59,7 +59,62 @@ ODK Central Frontend communicates with ODK Central Backend in part using a sessi
-- Enketo is a web form engine used to show form previews and allow for web-based data entry. The development `nginx.conf` is not configured to work with Enketo.
+- Enketo is a web form engine used to show form previews and allow for web-based data entry. The following section describes how to set up an Enketo server for use by ODK Central Frontend and ODK Central Backend.
+
+## (Optional) Running Enketo for development
+
+Enketo provides a web-based form submission UI, which is used in ODK Central for previewing forms, collecting new Submissions through Public Access Links, and editing Submissions. The following configuration is only required if you need this functionality in development. Enketo is automatically installed as part of the production ODK Central stack (via the [`central`](https://github.com/getodk/central) repository).
+
+Enketo runs as a Node.js server and caches intermediate representations of forms in Redis. ODK Central Backend stores information for managing and launching Enketo forms from ODK Central Frontend, so all three must be configured together.
+
+- Clone [enketo-express](https://github.com/enketo/enketo-express) and read the [Manual Installation Instructions](https://github.com/enketo/enketo-express). Specifically:
+	* Install Redis. (Node.js will already be installed to run ODK Central Frontend and Backend.) Enketo will expect Redis on the default port of `6379`.
+	* Install dependencies with `npm install`.
+- Configure Enketo. A minimal `config/config.json` for Enketo looks like:
+
+```
+{
+    "port": "8005",
+    "linked form and data server": {
+        "server url": "localhost:8989",
+        "api key": "enketorules",
+        "authentication": {
+            "allow insecure transport": "true"
+        }
+    },
+    "base path": "-",
+    "redis": {
+        "cache": {
+            "port": "6379"
+        }
+    }
+}
+```
+
+- Configure ODK Central Backend. Add the following to `config/default.json`.
+
+```
+    "enketo": {
+      "url": "http://localhost:8005/-",
+      "apiKey": "enketorules"
+    },
+```
+
+- Configure Nginx in ODK Central Frontend. Add the following to `nginx.conf` inside `server`, before or after `location /v1/`. This will create a reverse proxy for the Enketo server in the same way there is a reverse proxy to the ODK Central Backend server. You will need to restart Nginx after this.
+
+```
+    location /- {
+      proxy_pass http://localhost:8005/-;
+      proxy_redirect off;
+      proxy_set_header Host $host;
+    }
```

Comment:
I'm thinking it might be easiest for you to go ahead and put this in `nginx.conf`: I don't think there's any harm in always having it there. If Enketo isn't running, there'd be an error message, but that's also the case with the current `nginx.conf`. The production `nginx.conf` puts the Enketo `location` block before the other `location` blocks, so I think it'd work well to do that here as well.

Reply:
I agree that it feels fine to put it here in `nginx.conf`! Less sure on putting it in the backend, but your reasoning below sounds good!